### PR TITLE
update to 0.3.0

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by jakirkham** [comment]
Could you please add the license file portion back now that the associated change should have landed?

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender. :)

**Comment by sannykr** [comment]
@conda-forge/pytest-cookies : Could you please review and merge if everything looks okay

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ about:
   #
   # https://github.com/hackebrot/pytest-cookies/pull/10
   #
```
Comment (jakirkham) [inline recipe/meta.yaml]: This comment can be dropped as well. Was just referencing the removed license. When removing it feel free to add `[ci skip]` in the new commit message as we already know this passes and the change is non-actionable for the CI.

**Comment by sannykr** [comment]
@conda-forge/pytest-cookies : Could you please review and merge if everything looks okay.

**Comment by jakirkham** [comment]
Thanks @sannykr.

**Pull request merged**